README.md

```diff
@@ -138,6 +138,28 @@ end
 [ Info: Yo Dawg! it is all good
 ```
 
+### Respecting `maxlog` convention
+
+An `ActiveFilterLogger` can be used to wrap another logger to obey `maxlog` directives, for example,
+similar to the `make_throttled_logger` example below,
+```julia
+function make_maxlog_logger(logger)
+    counts = Dict{Symbol,Int}()
+    return ActiveFilteredLogger(logger) do log
```

Comment:
Since this is based on `id` I think it can use an `EarlyFilteredLogger`?

Reply:
I tried that at first, but the issue is we need `kwargs.maxlog` too

Reply:
Ah, right.